sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyEncryptionKeyClientBuilder.java

```diff
@@ -438,4 +453,20 @@ public KeyEncryptionKeyClientBuilder disableChallengeResourceVerification() {
 
         return this;
     }
+
+    /**
+     * Disables the ability to perform cryptographic operations locally, performing all cryptographic operations on the
```

Comment:
We should change the name to `disableKeyCaching` and we should update javadocs to mention that the JWT key is retrieved from the service and cached to perform local crypto operations.